components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -446,6 +446,70 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /applications/{applicationId}/exportFile:
+    get:
+      tags:
+        - Applications
+      operationId: exportApplicationAsFile
+      summary: |
+        Export application as an XML file
```

Comment:
Let's update these places - add all supporting file formats

Reply:
Updated both summary and description